monitoring/rid_qualifier/rid_qualifier_entry.py

```diff
@@ -0,0 +1,52 @@
+#!env/bin/python3
+
+import os
+import sys
+import argparse
+import asyncio
+from urllib.parse import urlparse
+import monitoring.rid_qualifier.test_executor as test_executor 
+
+def is_url(url_string):
+    try:
+        urlparse(url_string)
+    except ValueError:
+        raise ValueError("A valid injection_url must be passed")
+
+def parseArgs() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="Exceute RID_Qualifier for a locale")
+
+    parser.add_argument(
+        "--auth",
+        help="Auth spec for obtaining authorization to DSS instances; see README.md")
+
+    parser.add_argument(
+        "--locale",
+        help="A three letter ISO 3166 country code to run the qualifier against")
```

Comment:
nit: It would help the user to know that this country code should be the same one used to simulate the flight data in flight_data_generator

Reply:
Thanks! Modified now!